appeals/api/src/database/seed/LPAs/prod.js

```diff
@@ -0,0 +1,12 @@
+/** @typedef {Omit<import('@pins/appeals.api').Schema.LPA, 'id'>} LPA */
```

Comment:
We should use the create Prisma type here, it'll be something like

import('<db-client>').Prisma.LpaCreateInput